### PR TITLE
Add tolerations support for qstatefulset

### PR DESCRIPTION
**Pull request by rohitsakala**
[#170257319](https://www.pivotaltracker.com/n/projects/2192232/stories/170257319)

### CODE REVIEW

File: docs/controllers/quarks_statefulset.md [hunk 1]
```diff
@@ -187,6 +188,10 @@ If zones are set for an `QuarksStatefulSet`, the following occurs:
   AZ_INDEX="zone index"
   ```
 
+##### Tolerations
+
+Taints and tolerations is a concept defined in kubernetes to repel pods from nodes [0]. Defining tolerations is same as defined in the kubernetes docs. Keep in mind the affinity rules added by the controller when az's are defined. An example is specified in the examples folder.
```
Comment (viovanov) [inline docs/controllers/quarks_statefulset.md]:
```
Taints and tolerations are a concept defined in kubernetes to repel pods from nodes [0]. Defining tolerations is the same as defined in the kubernetes docs. Keep in mind the affinity rules added by the controller when az's are defined. An example is specified in the examples folder.
```

Comment (viovanov) [reply docs/controllers/quarks_statefulset.md]: Also, please don't use a bookmark `[0]` - just use a markdown link `[]()`

File: docs/from_bosh_to_kube.md [hunk 2]
```diff
@@ -307,6 +307,8 @@ instance_groups:
           # ImagePullSecrets is an optional list of references to secrets to use for pulling any of the images.
           # This field in PodSpec can be automated by setting the imagePullSecrets in a serviceAccount.
           ImagePullSecrets: {}
+          # Tolerations and taints is a concept defined in kubernetes to repel pods from nodes. [4]
```
Comment (viovanov) [inline docs/from_bosh_to_kube.md]: `Tolerations and taints are a concept defined in kubernetes`

Comment (rohitsakala) [reply docs/from_bosh_to_kube.md]: Done.

File: docs/controllers/quarks_statefulset.md (hunk 1 above)
Comment (rohitsakala) [reply docs/controllers/quarks_statefulset.md]: Done.

**Pull request merged**